seqr/views/utils/dataset_utils.py

```diff
@@ -155,6 +156,7 @@ def _find_or_create_missing_sample_records(
                 individual=individual,
                 created_date=timezone.now(),
                 is_active=create_active,
+                tissue_type=sample_id_to_tissue_type.get(sample_key[0], None) if sample_id_to_tissue_type else None,
```

Comment:
this needs to be by sample key not by sample ID, as otherwise we will get incorrect behavior for the same sample ID in multiple projects

Reply:
Can we assume the samples with the same sample ID for different projects are identical in the real world? It seems reasonable because the samples with the same sample ID look like the same sample but have been loaded and analyzed by multiple projects.

Reply:
No this is not acceptable